robottelo/decorators/__init__.py

```diff
@@ -32,8 +32,12 @@
 tier4 = pytest.mark.tier4
 # Destructive tests
 destructive = pytest.mark.destructive
-# Run for upgrade
+# Upgrade
 upgrade = pytest.mark.upgrade
+# Pre-Upgrade Scenarios
+pre_upgrade = pytest.mark.pre_upgrade
+# Post-Upgrade Scenarios
+post_upgrade = pytest.mark.post_upgrade
```

Comment:
If I understood correct above marker are added to manage at same way as others ... however all other scenarios are currently using as `from upgrade_tests import pre_upgrade, post_upgrade` .

Can we update in single shot for all scenarios instead of just add for one .. what do you think ?

Reply:
These markers are mostly being used in robottelo for upgrade scenarios, So lets keep it here and in single place.

We will just update all the test scenario later in separate PR.

Reply:
@jyejare as per my understanding we should not go with this change and use separate PR to avoid duplicate of markers .

Reply:
+1, Updated PR.

Reply:
Thanks , above is still exists , please check :)

Reply:
Done